### PR TITLE
Tutorial 10: Merge fixation and blink detection ids into gaze dataframe

### PR DESCRIPTION
**Pull request by papr**

### CODE REVIEW

**Comment by papr** [comment]
@n-m-t Please review this tutorial when you have time.

File: 10_apply_fixation_and_blink_detection_to_gaze.ipynb [hunk 1]
```diff
@@ -0,0 +1,520 @@
+{
```
Comment (N-M-T) [inline 10_apply_fixation_and_blink_detection_to_gaze.ipynb]: <p><s>- [...] detect blinks an fixations an how to export data.</s></p><p>+ [...] detect blinks and fixations an<strong>d</strong> how to export data.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pupil-labs/pupil-tutorials/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='0'/>

**Comment by N-M-T** [comment]
@papr new naming and description are great. The purpose of the tutorial is clear from the outset.

I do suggest directing the 'Pupil Player' link to the Player-specific section of the docs: https://docs.pupil-labs.com/core/software/pupil-player/#pupil-player

**Comment by papr** [comment]
Good catch re the outdated link! I fixed it as suggested. Thank you very much for your review!

**Pull request merged**